lib/sablon/image.rb

```diff
@@ -0,0 +1,17 @@
+module Sablon
+  class Image
+    include Singleton
+    attr_reader :definitions
+
+    Definition = Struct.new(:name, :data, :rid) do
+      def inspect
+        "#<Image #{name}:#{rid}"
+      end
+    end
+
+    def self.create_by_path(path, random = nil)
```

Comment:
Yeah, you seem right. I'll change that, it does not make sense to mix production and test code just for testing purpose